system/ee/ExpressionEngine/Addons/structure/views/module_settings.php

```diff
@@ -49,6 +49,24 @@
                 </select>
             </td>
         </tr>
+        <tr class="odd">
+            <td><?=lang('show_nav_status')?></td>
+            <td>
+                <select name="show_nav_status">
+                    <option value="y"<?=set_select('yes', 'Yes', ($settings['show_nav_status'] == '' || $settings['show_nav_status'] == 'y' ? 'y' : ''));?>><?=lang('yes')?></option>
```

Comment:
This defaults to "yes", but the rest of code does not seem to respect that. By default, the nav status is not show (which I think it should be) - however when I visit module settings, it shows as "yes", which is not correct